asv_bench/asv.conf.json

```diff
@@ -41,7 +41,10 @@
         "sqlalchemy": [],
         "scipy": [],
         "numexpr": [],
-        "pytables": [],
+        "tables": [],
```

Comment:
@qwhelan I see... That's weird that package is named ``tables`` in pypi. even original webpage reads http://www.pytables.org/